cliconfig/credentials/native_store_test.go

```diff
@@ -0,0 +1,170 @@
+// +build test_unit
```

Comment:
See:
https://github.com/docker/docker/pull/20107/files#diff-734ed399478ed3c45691f17639d62e46
and
https://github.com/docker/docker/pull/20107/files#diff-ac0f1eddc56b3465724336aa5a1a649a